lib/generators/rspec/mailer/templates/mailer_spec.rb

```diff
@@ -1,10 +1,10 @@
 require "rails_helper"
 
 <% module_namespacing do -%>
-RSpec.describe <%= class_name %><%= Rails.version.to_f >= 5.0 ? "Mailer" : "" %>, <%= type_metatag(:mailer) %> do
+RSpec.describe <%= class_name.gsub(/Mailer$/i, '') %><%= Rails.version.to_f >= 5.0 ? "Mailer" : "" %>, <%= type_metatag(:mailer) %> do
```

Comment:
I'd prefer it if we did this like this:

```
<%= Rails.version.to_f >= 5.0 ? class_name.gsub(/(Mailer)?$/i, 'Mailer') : class_name %>
```

As this will only run the modification on the edge case we know exists.